### PR TITLE
Update tidb-distributed-execution-framework.md

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/15602
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-distributed-execution-framework.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-title: TiDB Backend Task Distributed Execution Framework
-summary: Learn the use cases, limitations, usage, and implementation principles of the TiDB backend task distributed execution framework.
+title: TiDB Backend Task Distributed Execution Framework (DXF)
```
Comment (Frank945946) [inline tidb-distributed-execution-framework.md:2]:
```suggestion
title: TiDB Backend Task Distributed eXecution Framework (DXF)
```

File: tidb-distributed-execution-framework.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+summary: Learn the use cases, limitations, usage, and implementation principles of the TiDB backend task distributed execution  (DXF).
```
Comment (Frank945946) [inline tidb-distributed-execution-framework.md:3]:
```suggestion
summary: Learn the use cases, limitations, usage, and implementation principles of the TiDB backend task distributed execution framework (DXF).
```

**Review by Frank945946 — APPROVED** [review]
Rest is LGTM

File: tidb-distributed-execution-framework.md [hunk 3]
```diff
@@ -13,13 +13,9 @@ summary: Learn the use cases, limitations, usage, and implementation principles
...
+TiDB adopts a computing-storage separation architecture with excellent scalability and elasticity. Starting from v7.1.0, TiDB introduces a backend task DXF (Distributed eXecution Framework) to further leverage the resource advantages of the distributed architecture. The goal of this framework is to implement unified scheduling and distributed execution of all backend tasks, and to provide unified resource management capabilities for both overall and individual backend tasks, which better meets users' expectations for resource usage.
 
-This document describes the use cases, limitations, usage, and implementation principles of the TiDB backend task distributed execution framework.
-
-> **Note:**
->
-> This framework does not support the distributed execution of SQL queries.
+This document describes the use cases, limitations, usage, and implementation principles of the DXF.
```
Comment (Benjamin2037) [inline tidb-distributed-execution-framework.md:18]:
```suggestion
This document describes the use cases, usage, and implementation principles of the DXF.
```

File: tidb-distributed-execution-framework.md [hunk 4]
```diff
@@ -56,9 +52,13 @@ Currently, for TiDB Self-Hosted, the TiDB backend task distributed execution fra
 
 - `IMPORT INTO` is used to import data in formats such as `CSV`, `SQL`, and `PARQUET` into an empty table. For more information, see [`IMPORT INTO`](https://docs.pingcap.com/tidb/v7.2/sql-statement-import-into).
 
+## Limitation
+
+The DXF can only schedule one `ADD INDEX` task at a time. If a second `ADD INDEX` task is submitted before the first `ADD INDEX` task is completed, the second `ADD INDEX` task is executed through a transaction instead of the DXF.
```
Comment (hfxsd) [inline tidb-distributed-execution-framework.md:57]:
```suggestion
The DXF can only schedule the distributed execution of one `ADD INDEX` task at a time. If a new `ADD INDEX` task is submitted before the current `ADD INDEX` distributed task has finished, the new task is executed through a transaction.
```

File: tidb-distributed-execution-framework.md [hunk 5]
```diff
@@ -39,13 +35,13 @@ In a database management system, in addition to the core transactional processin
...
 
-Enabling the TiDB backend task distributed execution framework can solve the above problems and has the following three advantages:
+Enabling the DXF can solve the above problems and has the following three advantages:
 
 - The framework provides unified capabilities for high scalability, high availability, and high performance.
 - The framework supports distributed execution of backend tasks, which can flexibly schedule the available computing resources of the entire TiDB cluster, thereby better utilizing the computing resources in a TiDB cluster.
 - The framework provides unified resource usage and management capabilities for both overall and individual backend tasks.
 
-Currently, for TiDB Self-Hosted, the TiDB backend task distributed execution framework supports the distributed execution of the `ADD INDEX` and `IMPORT INTO` statements. For TiDB Cloud, the `IMPORT INTO` statement is not applicable.
+Currently, for TiDB Self-Hosted, the DXF supports the distributed execution of the `ADD INDEX` and `IMPORT INTO` statements. For TiDB Cloud, the `IMPORT INTO` statement is not applicable until v7.2.
```
Comment (qiancai) [inline tidb-distributed-execution-framework.md:44]: The `IMPORT INTO` statement is not supported on TiDB Cloud.

File: tidb-distributed-execution-framework.md [hunk 6]
```diff
@@ -106,28 +106,25 @@ Adjust the following system variables related to Fast Online DDL:
 
     </CustomContent>
 
-2. Adjust the following system variables that might affect the distributed execution of DDL tasks according to your needs:
+2. It is recommended that you use their default values for the following system variables that might affect the distributed execution of DDL tasks:
```
Comment (qiancai) [inline tidb-distributed-execution-framework.md:109]:
```suggestion
2. For the following system variables that might affect the distributed execution of DDL tasks, it is recommended that you use their default values:
```

File: tidb-distributed-execution-framework.md [hunk 7]
```diff
@@ -39,13 +35,13 @@ In a database management system, in addition to the core transactional processin
 - Might need to be executed periodically, but at a low frequency.
 - If the resources are not properly controlled, they are prone to affect TP and AP tasks, lowering the database service quality.
 
-Enabling the TiDB backend task distributed execution framework can solve the above problems and has the following three advantages:
+Enabling the DXF can solve the above problems and has the following three advantages:
 
 - The framework provides unified capabilities for high scalability, high availability, and high performance.
 - The framework supports distributed execution of backend tasks, which can flexibly schedule the available computing resources of the entire TiDB cluster, thereby better utilizing the computing resources in a TiDB cluster.
 - The framework provides unified resource usage and management capabilities for both overall and individual backend tasks.
```
Comment (qiancai) [inline tidb-distributed-execution-framework.md:42]:
```suggestion
- The framework provides unified resource usage and management capabilities for both overall and individual tasks.
```

File: tidb-distributed-execution-framework.md [hunk 8]
```diff
@@ -39,13 +35,13 @@ In a database management system, in addition to the core transactional processin
 - Might need to be executed periodically, but at a low frequency.
 - If the resources are not properly controlled, they are prone to affect TP and AP tasks, lowering the database service quality.
 
-Enabling the TiDB backend task distributed execution framework can solve the above problems and has the following three advantages:
+Enabling the DXF can solve the above problems and has the following three advantages:
 
 - The framework provides unified capabilities for high scalability, high availability, and high performance.
 - The framework supports distributed execution of backend tasks, which can flexibly schedule the available computing resources of the entire TiDB cluster, thereby better utilizing the computing resources in a TiDB cluster.
```
Comment (qiancai) [inline tidb-distributed-execution-framework.md:41]:
```suggestion
- The framework supports distributed execution of tasks, which can flexibly schedule the available computing resources of the entire TiDB cluster, thereby better utilizing the computing resources in a TiDB cluster.
```

File: tidb-distributed-execution-framework.md [hunk 9]
```diff
@@ -13,39 +13,48 @@ summary: Learn the use cases, limitations, usage, and implementation principles
...
 - If the resources are not properly controlled, they are prone to affect TP and AP tasks, lowering the database service quality.
 
-Enabling the TiDB backend task distributed execution framework can solve the above problems and has the following three advantages:
+Enabling the DXF can solve the above problems and has the following three advantages:
 
 - The framework provides unified capabilities for high scalability, high availability, and high performance.
-- The framework supports distributed execution of backend tasks, which can flexibly schedule the available computing resources of the entire TiDB cluster, thereby better utilizing the computing resources in a TiDB cluster.
-- The framework provides unified resource usage and management capabilities for both overall and individual backend tasks.
+- The framework supports distributed execution of tasks, which can flexibly schedule the available computing resources of the entire TiDB cluster, thereby better utilizing the computing resources in a TiDB cluster.
+- The framework provides unified resource usage and management capabilities for both overall and individual tasks.
```
Comment (Frank945946) [inline tidb-distributed-execution-framework.md:42]:
```suggestion
- The DXF supports distributed execution of tasks, which can flexibly schedule the available computing resources of the entire TiDB cluster, thereby better utilizing the computing resources in a TiDB cluster.
- The DXF provides unified resource usage and management capabilities for both overall and individual tasks.
```

File: tidb-distributed-execution-framework.md [hunk 10]
```diff
@@ -13,39 +13,48 @@ summary: Learn the use cases, limitations, usage, and implementation principles
...
+
+<CustomContent platform="tidb-cloud">
+
+Currently, for TiDB Self-Hosted, the DXF supports the distributed execution of the `ADD INDEX`. `ADD INDEX` is a DDL statement used to create indexes. For example:
```
Comment (Frank945946) [inline tidb-distributed-execution-framework.md:46]:
```suggestion
Currently, the DXF supports the distributed execution of the `ADD INDEX`. `ADD INDEX` is a DDL statement used to create indexes. For example:
```

File: tidb-distributed-execution-framework.md [hunk 11]
```diff
@@ -13,39 +13,48 @@ summary: Learn the use cases, limitations, usage, and implementation principles
...
+ALTER TABLE t1 ADD INDEX idx1(c1);
+CREATE INDEX idx1 ON table t1(c1);
+```
+
+</CustomContent>
 
-Currently, for TiDB Self-Hosted, the TiDB backend task distributed execution framework supports the distributed execution of the `ADD INDEX` and `IMPORT INTO` statements. For TiDB Cloud, the `IMPORT INTO` statement is not applicable.
+<CustomContent platform="tidb">
+
+Currently, for TiDB Self-Hosted, the DXF supports the distributed execution of the `ADD INDEX` and `IMPORT INTO` statements. For TiDB Cloud, the `IMPORT INTO` statement is not applicable until v7.2.
```
Comment (Frank945946) [inline tidb-distributed-execution-framework.md]:
```suggestion
Currently, for TiDB Self-Hosted, the DXF supports the distributed execution of the `ADD INDEX` and `IMPORT INTO` statements.
```

Comment (Frank945946) [reply tidb-distributed-execution-framework.md]: However, it is duplicated with line 46

File: tidb-distributed-execution-framework.md (hunk 5 above)
Comment (hfxsd) [reply tidb-distributed-execution-framework.md]: Updated

File: tidb-distributed-execution-framework.md [hunk 12]
```diff
@@ -106,33 +121,30 @@ Adjust the following system variables related to Fast Online DDL:
 
     </CustomContent>
 
-2. Adjust the following system variables that might affect the distributed execution of DDL tasks according to your needs:
+2. For the following system variables that might affect the distributed execution of DDL tasks, it is recommended that you use their default values:
```
Comment (Frank945946) [inline tidb-distributed-execution-framework.md:124]:
```suggestion
2. In general, for the following system variables that might affect the distributed execution of DDL tasks, it is recommended that you use their default values:
```

File: tidb-distributed-execution-framework.md [hunk 13]
```diff
@@ -106,33 +121,30 @@ Adjust the following system variables related to Fast Online DDL:
...
+3. Starting from v7.4.0, you can adjust the number of TiDB nodes that perform background tasks according to actual needs. After deploying a TiDB cluster, you can set the instance-level system variable [`tidb_service_scope`](/system-variables.md#tidb_service_scope-new-in-v740) for each TiDB node in the cluster. When `tidb_service_scope` of a TiDB node is set to `background`, the TiDB node can execute background tasks. When `tidb_service_scope` of a TiDB node is set to the default value "", the TiDB node cannot execute background tasks. If `tidb_service_scope` is not set for any TiDB node in a cluster, the DXF schedules all TiDB nodes to execute background tasks by default.
```
Comment (Frank945946) [inline tidb-distributed-execution-framework.md:131]:
```suggestion
3. Starting from v7.4.0, you can adjust the number of TiDB nodes that perform the DXF tasks according to actual needs. After deploying a TiDB cluster, you can set the instance-level system variable [`tidb_service_scope`](/system-variables.md#tidb_service_scope-new-in-v740) for each TiDB node in the cluster. When `tidb_service_scope` of a TiDB node is set to `background`, the TiDB node can execute the DXF tasks. When `tidb_service_scope` of a TiDB node is set to the default value "", the TiDB node cannot execute DXF tasks. If `tidb_service_scope` is not set for any TiDB node in a cluster, the DXF schedules all TiDB nodes to execute background tasks by default.
```

File: TOC.md [hunk 14]
```diff
@@ -644,6 +644,9 @@
     - [Data Validation](/tiflash/tiflash-data-validation.md)
     - [Compatibility](/tiflash/tiflash-compatibility.md)
     - [Pipeline Execution Model](/tiflash/tiflash-pipeline-model.md)
+  - TiDB Distributed Execution Framework
```
Comment (qiancai) [inline TOC.md:647]: 这里要写成 eXecution 吗

File: TOC-tidb-cloud.md
Comment (qiancai) [inline TOC-tidb-cloud.md:1]: 是否需要进行和 TOC.md 一样的改动

File: tidb-distributed-execution-framework.md [hunk 15]
```diff
@@ -88,7 +103,7 @@ Adjust the following system variables related to Fast Online DDL:
 
 ## Usage
 
-1. To enable the distributed framework, set the value of [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710) to `ON`:
+1. To enable the DXF, set the value of [`tidb_enable_dist_task`](/system-variables.md#tidb_enable_dist_task-new-in-v710) to `ON`:
 
     ```sql
```
Comment (qiancai) [inline tidb-distributed-execution-framework.md:108]: L114 和 L120 中的 background tasks 需要修改为 tasks

File: releases/release-6.6.0.md [hunk 16]
```diff
@@ -77,9 +77,9 @@ In v6.6.0-DMR, the key new features and improvements are as follows:
 
     For more information, see [documentation](/partitioned-raft-kv.md).
 
-* Support the distributed parallel execution framework for DDL operations (experimental) [#37125](https://github.com/pingcap/tidb/issues/37125) @[zimulala](https://github.com/zimulala)
+* Support the TiDB Distributed eXecution Framework (DXF) for DDL operations (experimental) [#37125](https://github.com/pingcap/tidb/issues/37125) @[zimulala](https://github.com/zimulala)
```
Comment (hfxsd) [inline releases/release-6.6.0.md:80]:
```suggestion
* Support the distributed parallel execution framework for DDL operations (experimental) [#37125](https://github.com/pingcap/tidb/issues/37125) @[zimulala](https://github.com/zimulala)
```

File: releases/release-6.6.0.md [hunk 17]
```diff
@@ -77,9 +77,9 @@ In v6.6.0-DMR, the key new features and improvements are as follows:
...
+    In previous versions, only one TiDB instance in the entire TiDB cluster was allowed to handle schema change tasks as a DDL owner. To further improve DDL concurrency for large table's DDL operations, TiDB v6.6.0 introduces the TiDB Distributed eXecution Framework (DXF) for DDL, through which all TiDB instances in the cluster can concurrently execute the `StateWriteReorganization` phase of the same task to speed up DDL execution. This feature is controlled by the system variable [`tidb_ddl_distribute_reorg`](https://docs.pingcap.com/tidb/v6.6/system-variables#tidb_ddl_distribute_reorg-new-in-v660) and is currently only supported for `Add Index` operations.
```
Comment (hfxsd) [inline releases/release-6.6.0.md:82]:
```suggestion
    In previous versions, only one TiDB instance in the entire TiDB cluster was allowed to handle schema change tasks as a DDL owner. To further improve DDL concurrency for large table's DDL operations, TiDB v6.6.0 introduces the distributed parallel execution framework for DDL, through which all TiDB instances in the cluster can concurrently execute the `StateWriteReorganization` phase of the same task to speed up DDL execution. This feature is controlled by the system variable [`tidb_ddl_distribute_reorg`](https://docs.pingcap.com/tidb/v6.6/system-variables#tidb_ddl_distribute_reorg-new-in-v660) and is currently only supported for `Add Index` operations.
```

File: releases/release-7.5.0.md [hunk 18]
```diff
@@ -32,7 +32,7 @@ Compared with the previous LTS 7.1.0, 7.5.0 includes new features, improvements,
...
+    <td>TiDB v7.2.0 introduced the <a href="https://docs.pingcap.com/tidb/v7.5/tidb-distributed-execution-framework" target="_blank">Distributed eXecution Framework (DXF)</a>. For tasks that take advantage of this framework, v7.4 introduces global sorting to eliminate the unnecessary I/O, CPU, and memory spikes caused by temporarily out-of-order data during data re-organization tasks. The global sorting takes advantage of external shared object storage (Amazon S3 in this first iteration) to store intermediary files during the job, adding flexibility and cost savings. Operations like <code>ADD INDEX</code> and <code>IMPORT INTO</code> will be faster, more resilient, more stable, more flexible, and cost less to run.</td>
```
Comment (hfxsd) [inline releases/release-7.5.0.md:35]:
```suggestion
    <td>TiDB v7.1.0 introduced the <a href="https://docs.pingcap.com/tidb/v7.5/tidb-distributed-execution-framework" target="_blank">Distributed eXecution Framework (DXF)</a>. For tasks that take advantage of this framework, v7.4 introduces global sorting to eliminate the unnecessary I/O, CPU, and memory spikes caused by temporarily out-of-order data during data re-organization tasks. The global sorting takes advantage of external shared object storage (Amazon S3 in this first iteration) to store intermediary files during the job, adding flexibility and cost savings. Operations like <code>ADD INDEX</code> and <code>IMPORT INTO</code> will be faster, more resilient, more stable, more flexible, and cost less to run.</td>
```

File: releases/release-7.5.0.md [hunk 19]
```diff
@@ -71,19 +71,19 @@ Compared with the previous LTS 7.1.0, 7.5.0 includes new features, improvements,
...
+    The DXF introduced in v6.6.0 has become GA. In versions before TiDB v7.1.0, only one TiDB node can execute DDL tasks at the same time. Starting from v7.1.0, multiple TiDB nodes can execute the same DDL task in parallel under the DXF. Starting from v7.2.0, the DXF supports multiple TiDB nodes to execute the same `IMPORT INTO` task in parallel, thereby better utilizing the resources of the TiDB cluster and significantly improving the performance of DDL and `IMPORT INTO` tasks. In addition, you can also increase TiDB nodes to linearly improve the performance of these tasks.
```
Comment (hfxsd) [inline releases/release-7.5.0.md:84]:
```suggestion
    The DXF introduced in v7.1.0 has become GA. In versions before TiDB v7.1.0, only one TiDB node can execute DDL tasks at the same time. Starting from v7.1.0, multiple TiDB nodes can execute the same DDL task in parallel under the DXF. Starting from v7.2.0, the DXF supports multiple TiDB nodes to execute the same `IMPORT INTO` task in parallel, thereby better utilizing the resources of the TiDB cluster and significantly improving the performance of DDL and `IMPORT INTO` tasks. In addition, you can also increase TiDB nodes to linearly improve the performance of these tasks.
```

File: tidb-roadmap.md [hunk 20]
```diff
@@ -28,8 +28,8 @@ In the course of development, this roadmap is subject to change based on user ne
       <td>
         <ul>
           <li>
-            <b>Distributed execution framework</b><br /><i>
-            In v7.2.0, TiDB introduced the distributed execution framework for background tasks (such as DDL and analyze). This is the foundation for parallelizing these tasks across compute nodes. v7.4.0 introduces global sorting in distributed re-organization tasks (such as DDL and import), which greatly mitigates extra resource consumption in storage. Optionally, external shared storage can be leveraged for simplicity and cost savings.</i>
+            <b>Distributed eXecution Framework (DXF)</b><br /><i>
+            In v7.2.0, TiDB introduced the DXF for tasks (such as DDL and analyze). This is the foundation for parallelizing these tasks across compute nodes. v7.4.0 introduces global sorting in distributed re-organization tasks (such as DDL and import), which greatly mitigates extra resource consumption in storage. Optionally, external shared storage can be leveraged for simplicity and cost savings.</i>
```
Comment (hfxsd) [inline tidb-roadmap.md:32]:
```suggestion
            In v7.1.0, TiDB introduced the DXF for tasks (such as DDL and analyze). This is the foundation for parallelizing these tasks across compute nodes. v7.4.0 introduces global sorting in distributed re-organization tasks (such as DDL and import), which greatly mitigates extra resource consumption in storage. Optionally, external shared storage can be leveraged for simplicity and cost savings.</i>
```

File: tidb-distributed-execution-framework.md [hunk 21]
```diff
@@ -1,9 +1,9 @@
 ---
```
[the remark this answers is not in the document]
Comment (hfxsd) [reply tidb-distributed-execution-framework.md:1]: Added

**Pull request merged**